frontend/rust-lib/dart-ffi/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 name = "dart_ffi"
 # this value will change depending on the target os
 # default static lib
-crate-type = ["staticlib"]
+crate-type = ["cdylib"]
```

Comment:
Don't modify this line.

Reply:
I didn't modified it by myself. I think got modified automatically.